SYNTAX.md

```diff
@@ -49,6 +49,12 @@ The following tables will give you an overview of the placeholders:
 | `${table_name}`      | The name of the table this constraint affects             |
 | `${column_names}`    | A comma-separated list of columns this constraint affects |
 
+### Placeholders for Type-Mappings
```

Comment:
In der SYNTAX.md fehlt eine Beschreibung der Type Mappings

Reply:
👍